testsuite/.rubocop_todo.yml

```diff
@@ -225,6 +225,7 @@ Lint/NumberConversion:
     - 'features/step_definitions/navigation_steps.rb'
     - 'features/step_definitions/salt_steps.rb'
     - 'features/step_definitions/setup_steps.rb'
+    - 'features/step_definitions/vm_steps.rb'
```

Comment:
That looks like another Cop candidate to just disable.

Reply:
I fixed it by looking at the docs for that:

Old
```ruby
 == count.to_i
```

New
```ruby
 == count { |i| Integer(i) }
```

Reply:
Apparently this did not work as expected:

```bash
/step_definitions/vm_steps.rb:212
    And "test-vm2" virtual machine on "kvm_server" should have 1 NIC using "test-net0" network                                            # features/step_definitions/vm_steps.rb:235
      undefined method `count' for #<Object:0x000056524b107478> (NoMethodError)
      ./features/step_definitions/vm_steps.rb:239:in `block (2 levels) in <top (required)>'
      ./features/support/commonlib.rb:113:in `block in repeat_until_timeout'
      ./features/support/commonlib.rb:102:in `repeat_until_timeout'
      ./features/step_definitions/vm_steps.rb:237:in `/^"([^"]*)" virtual machine on "([^"]*)" should have ([0-9]*) NIC using "([^"]*)" network$/'
      features/secondary/minkvm_guests.feature:190:in `"test-vm2" virtual machine on "kvm_server" should have 1 NIC using "test-net0" network'
```
Reverting it.